### ISSUE TITLE
[HOLD for payment 2024-07-30][Simplified Collect] Workspaces - Display picture/Avatar isn't shown in Workspace share Code page

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#27288]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. go settings -> workspaces -> new workspace
2. Now click the display avatar -> edit photo -> upload photo
3. Notice that the uploaded photo is displayed.
4. Now go to #announce or #admin or workspace chat of the same newly created workspace.
5. go to details page.
6. Notice that  there is 'expensify' symbol in QR code. i.e display picture of corresponding workspace isn't shown.

## Expected Result:
 display picture of corresponding workspace should be  shown in share code page.

## Actual Result:
display picture of corresponding workspace isn't shown in share code page for workspace.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** v1.3.68-15

**Reproducible in staging?:** Y

**Reproducible in production?:** Y

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/115492554/043ba9f9-cf8e-49ec-87af-564c63b459ba

https://github.com/Expensify/App/assets/115492554/a54c8906-fc54-4bf0-ba3e-8d443afaca92

**Expensify/Expensify Issue URL:**

**Issue reported by:** @ashimsharma10

**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1694259150016599

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01f242a32573a62742</li>
        <li>Upwork Job ID: 1701685406614237184</li>
        <li>Last Price Increase: 2023-10-03</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @abekkala</details>

**Comment by abekkala** [comment]
@sobitneupane what do you think of the proposal that has come in?

**Comment by sobitneupane** [comment]
@abekkala I'm uncertain whether not displaying avatars in QRCode for Workspace is a bug or an intentional design choice. Can you please confrim if showing avatars is the desired behavior?

**Comment by s-alves10** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

Workspace avatar isn't shown in share-code page of the workspace

### What is the root cause of that problem?

https://github.com/Expensify/App/blob/2ea1c71f9ea5d47b227126c8f2cf792300f0d9db/src/pages/ShareCodePage.js#L70

In the above code, `isReport` is true for workspace rooms and chats.
This is the root cause

### What changes do you think we should make in order to solve the problem?

We need to check if the report is workspace rooms or chat

```
        const isWorkspace = isReport && (ReportUtils.isAdminRoom(this.props.report) || ReportUtils.isAnnounceRoom(this.props.report) || ReportUtils.isPolicyExpenseChat(this.props.report));
```

And then get the proper logo as below

```
        let logo = UserUtils.getAvatarUrl(this.props.currentUserPersonalDetails.avatar, this.props.currentUserPersonalDetails.accountID);
        if (isReport) {
            if (isWorkspace) {
                logo = ReportUtils.getWorkspaceAvatar(this.props.report) || expensifyLogo;
            } else {
                logo = expensifyLogo;
            }
        }
```
<details>
    <summary>Result</summary>

![image](https://github.com/Expensify/App/assets/126839717/aeabc173-1d3f-4af0-b41b-40575dfdfa60)

</details>


### What alternative solutions did you explore? (Optional)

**Comment by abekkala** [comment]
Yes - the avatar should show
https://github.com/Expensify/App/issues/20993

@deetergp it looks like we used to show the avatar but not anymore.

**Comment by abekkala** [comment]
@sobitneupane is there a proposal here that you agree with to move forward with the fix?

**Comment by sobitneupane** [comment]
@abekkala Thank you for providing the update. Just to clarify, are we aiming to display the workspace avatar, which could be either user-uploaded or the default avatar, for every room(either default or user-created) on the Share Code Page.

**Comment by sobitneupane** [comment]
The proposals we have received so far displays workspace avatar on the Share Code Page only if it is user uploaded image. In case of default avatar, `expensifyLogo` is used.

**Comment by abekkala** [comment]
> @abekkala Thank you for providing the update. Just to clarify, are we aiming to display the workspace avatar, which could be either user-uploaded or the default avatar, for every room(either default or user-created) on the Share Code Page.

Ah, good clarification. I believe the answer is yes. @deetergp just want to get a second opinion.

**Comment by s-alves10** [comment]
In order to show default avatar of workspace rooms in share code page, I think we need to have URLs for [these default avatars](https://github.com/Expensify/App/blob/main/src/components/Icon/WorkspaceDefaultAvatars.js) because the `react-native-qrcode-svg` library doesn't support component type logo.
Another option is to make some changes to the library to customize the logo

**Comment by abekkala** [comment]
@deetergp maybe you can weigh in here? https://github.com/Expensify/App/issues/27288#issuecomment-1732694706

**Comment by deetergp** [comment]
> @abekkala Thank you for providing the update. Just to clarify, are we aiming to display the workspace avatar, which could be either user-uploaded or the default avatar, for every room(either default or user-created) on the Share Code Page.

I agree that this seems like the right behavior.

**Comment by sobitneupane** [comment]
@ZhenjaHorbach @s-alves10 If you are still interested, please consider [this](https://github.com/Expensify/App/issues/27288#issuecomment-1743613313) as expected behavior.

**Comment by s-alves10** [comment]
@sobitneupane 

Did you check https://github.com/Expensify/App/issues/27288#issuecomment-1736832823? Please let me know your thought on that

**Comment by sobitneupane** [comment]
Sorry for the delay.

I will try to review before EOD tomorrow (NPT).

**Comment by sobitneupane** [comment]
> In order to show default avatar of workspace rooms in share code page, I think we need to have URLs for [these default avatars](https://github.com/Expensify/App/blob/main/src/components/Icon/WorkspaceDefaultAvatars.js) because the react-native-qrcode-svg library doesn't support component type logo.
Another option is to make some changes to the library to customize the logo

How are we handling user default avatars? We should follow similar approach here as well.

**Comment by sobitneupane** [comment]
> it looks like we used to show the avatar but not anymore.

@abekkala We were not showing avatars from the beginning for reports. We used to show [room.png](https://github.com/Expensify/App/pull/18636/files#diff-3da9f786e38061bb1aaf203fd5d79a4922d7a59da8b764a4b31ae9b4189d1f8c) for reports (added in this [PR](https://github.com/Expensify/App/pull/18636)) which was later replaced by ExpensifyLogo.

**Comment by abekkala** [comment]
@sobitneupane when reviewing earlier discussions we _want_ to show the workspace avatar. Either an user-uploaded or default avatar, for every room(either default or user-created) on the Share Code Page.

**Comment by sobitneupane** [comment]
@s-alves10 https://github.com/Expensify/App/issues/27288#issuecomment-1754796310

**Comment by s-alves10** [comment]
@sobitneupane 

> How are we handling user default avatars? We should follow similar approach here as well.

We have URLs for user's default avatars

**Comment by sobitneupane** [comment]
Started discussion in [slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1697098846087309).

**Comment by abekkala** [comment]
@sobitneupane checking in again - do you feel there is a proposal here that will work? 
If not, I can pull in someone to get a fix out for this

**Comment by situchan** [comment]
@sobitneupane is OOO this week

**Comment by abekkala** [comment]
@sobitneupane are you back to review the proposals on this one? Do you need me to pull someone else in?

**Comment by s-alves10** [comment]
@sobitneupane 

[My latest proposal](https://github.com/Expensify/App/issues/27288#issuecomment-1723178639) can solve the issue without adding new URLs for workspace default avatars. Please check

**Comment by sobitneupane** [comment]
Sorry for the delay here. I was OOO. I will try to get to it in the weekend.

**Comment by abekkala** [comment]
@sobitneupane any update here?

**Comment by sobitneupane** [comment]
@s-alves10 I believe the better idea is to use url for workspace avatars if that is absolutely necessary. 
```js
 /* an image source object. example {uri: 'base64string'} or {require('pathToImage')} */
  logo?: ImageSourcePropType;
  ```
  If the avatars are locally available, can we use require('pathToImage')?
  
In the meantime, I will try to get more people involved in the [Slack discussion](https://expensify.slack.com/archives/C01GTK53T8Q/p1697098846087309)

**Comment by s-alves10** [comment]
> If the avatars are locally available, can we use require('pathToImage')?

We can't use it. That returns a component. That's why I suggested to patch the library

**Comment by abekkala** [comment]
@sobitneupane is the proposal sufficient? If not, I can pull someone in to get this one fixed asap

**Comment by sobitneupane** [comment]
> > If the avatars are locally available, can we use require('pathToImage')?
> 
> We can't use it. That returns a component. That's why I suggested to patch the library

@s-alves10 Rather than a patch, can we propose the upgrade in the upstream repository react-native-qr-code-svg?

**Comment by s-alves10** [comment]
Of course, we can create a PR there, but it looks like the last PR was merged in Feb 2023 and there are 10 pending PRs

**Comment by sobitneupane** [comment]
We are still having [discussion](https://expensify.slack.com/archives/C01GTK53T8Q/p1697098846087309) in slack. @s-alves10 Please feel free to add your thoughts in the slack disucssion.

I think we don't encourage to apply patch. So, if possible, let's look for other solutions.

**Comment by abekkala** [comment]
@sobitneupane do we have an acceptable fix for this? If not, I'll be pulling someone in today that can get this one moving. 

Thanks!

**Comment by grgia** [comment]
Followed up, it sounds like they are interested

> Hi Georgia and Expensify, thanks for showing the interest. I'm currently not available to deal with the taking over. May I contact you in a week or two?

Should we hold this issue that long? 

cc @roryabraham @sobitneupane

**Comment by sobitneupane** [comment]
Not a critical issue. So, I believe we can wait or a week or two.

**Comment by grgia** [comment]
Making weekly while we wait for reply from lib owner

**Comment by grgia** [comment]
followed up with the repo owner

**Comment by abekkala** [comment]
@grgia I'm going ooo though Jan 03. 
I don't _think_ there will be much movement through then so I'm keeping myself assigned as by the time I return this probably won't be at the payment stage for me to take action.
However, if things progress quickly you can reapply the Bug label to get someone one this to submit payment.
(I'll also be checking in on my GH's while I'm ooo)

**Comment by abekkala** [comment]
@grgia I know this one wasn't considered critical but are we still moving forward with this?

**Comment by grgia** [comment]
@abekkala I'm still discussing details via email with repo owner

**Comment by grgia** [comment]
bumped one final time, will call it and decide next steps pending next week

**Comment by abekkala** [comment]
@grgia is there any update for this one?

**Comment by grgia** [comment]
Yes, I'm in contact with the repo owner

**Comment by abekkala** [comment]
@grgia any update on moving forward here?

**Comment by grgia** [comment]
Making monthly, I think this is the way we should handle this, but it's not that high priority. Making monthly until we take over the repo

**Comment by abekkala** [comment]
@grgia - because this is internal and you've been handling: 

- Is this truly a high enough priority / ROI bug to fix?
- Will this affect more than x # of users?
- Does it affect users who pay us?
- How much of an edge case is it?
- If we fix the bug later, in a year or two, will that be OK?

Maybe this can simply be closed for now?

**Comment by abekkala** [comment]
@grgia given the age of this one - do you think this can simply be closed for now in favor of higher prioritized issues?

**Comment by trjExpensify** [comment]
Related discussion here: https://expensify.slack.com/archives/C036QM0SLJK/p1710800681715819?thread_ts=1710800218.107499&cid=C036QM0SLJK

**Comment by Kicu** [comment]
As discussed on slack I will be taking a look on this one and will prepare a PoC to push this forward

**Comment by grgia** [comment]
Thanks @Kicu !

**Comment by abekkala** [comment]
@kevinksullivan I'm ooo until April 08 - I'll be taking back any Open issues once I return

___

### CURRENT STATUS:
@Kicu will be working on a fix (no payment required)
PR Review will be done by @sobitneupane (payment will be done via NEWDOT)

**Comment by grgia** [comment]
@Kicu would you mind re-summarizing where you're at with this on the GH?

**Comment by Kicu** [comment]
here's my understanding after reading this whole (quite long 😅) discussion:

## Original issue/bug
The original issue is that if a user enters the details of a channel like `#announce` or `#admins` and clicks on "Share" we expect to display workspace avatar inside the QR code, but we instead display expensify logo.

## Root cause of that problem
In my mind there are 2 roots causes here:
 - on channel(report) Details Page we do not try to display the workspace avatars at all (we only display expensify logo) -  this can be easily fixed **but only** for user uploaded images,
 - [**this is main problem**] the library that we use to generate QR codes (https://github.com/awesomejerry/react-native-qrcode-svg) does not support providing SVG format for the logo image, and the default workspace logos are SVGs in our code.

According to this discussion on slack: https://swmansion.slack.com/archives/C036QM0SLJK/p1710800218107499 and comments in this issue we would like to take over the library and update it to allow displaying SVG logos.

## Planned next steps
 - I have forked https://github.com/awesomejerry/react-native-qrcode-svg and I will work to provide a PoC PR showing how we could implement displaying SVGs - so that we can agree on the solution. Hopefully soon expensify will be the owner of the repo
 - [OPTIONAL] there is a faster fix that would allow to display just the user uploaded workspace avatar for now, because that already is already done in the app. If you go to Profile > workspaces > workspace Page and click "Share" - there it works for user uploaded images, but not for SVGs. Not sure if we want to do this partial fix while we wait for the fix in the library? @grgia 

### Notes
 - this problem is not present for user avatars, because someone uploaded a `.png` version of user default avatars to a cdn and that is what is used for User share page;
 - uploading workspace default avatars as `png` is not feasible as there are too many

**Comment by luacmartins** [comment]
Thanks @Kicu! Looking forward to the POC.

>on channel(report) Details Page we do not try to display the workspace avatars at all (we only display expensify logo) - this can be easily fixed but only for user uploaded images,

I think we'd update this once we have a working solution for the SVGs, so the solution is consistent.

**Comment by Kicu** [comment]
Alright, here is some PoC code:

1. PR to my fork of `react-native-qrcode-svg`: https://github.com/Kicu/react-native-qrcode-svg/pull/1 - these are the changes required to display svg inside QR code, feel free to review and tell me if it makes sense
2. draft PR to Expensify app that uses ☝️ to actually display the svgs: https://github.com/Expensify/App/pull/39072 - but for now I only updated Workspace sharing, can do more tomorrow

Both require more polishing but I wanted to quickly see that this is possible and not use too much time on misc polish.
Also I'm not sure how close we are to actually owning the library.

Please tell me what next steps do you want me to take.

**Comment by luacmartins** [comment]
Nice one! @sobitneupane could you please review https://github.com/Kicu/react-native-qrcode-svg/pull/1 when you get a chance. Also we might wanna apply that fix to our fork of the repo once ready.

**Comment by Kicu** [comment]
Just wanted to stress out that the draft PR in Expensify is not 100% done, but it shows that the svg's are doable, so perhaps only a quick look is enough to tell me that this direction is correct.

Also in the long run what are the chances of actually having ownership of `react-native-qrcode-svg` do you guys think? There are some more updates that could be done there (rewrite it to TS is one)

**Comment by sobitneupane** [comment]
Thanks for the update @Kicu I believe we are going towards the right direction. Only improvement I could see is making use of existing `logo` prop if possible, else everything looks great. 

<details>
<summary>iOS: Native</summary>
<img width="332" alt="Screenshot 2024-03-28 at 13 51 24" src="https://github.com/Expensify/App/assets/25876548/c2182a71-987a-4887-b52d-605af73d6aa5">
<img width="332" alt="Screenshot 2024-03-28 at 13 51 47" src="https://github.com/Expensify/App/assets/25876548/e696cb2f-74b2-4f14-89d1-b21e34a62aab">



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>
<img width="372" alt="Screenshot 2024-03-28 at 13 29 08" src="https://github.com/Kicu/react-native-qrcode-svg/assets/25876548/255e2459-0c16-44c1-8361-6889973e7fc1">
<img width="372" alt="Screenshot 2024-03-28 at 13 29 14" src="https://github.com/Kicu/react-native-qrcode-svg/assets/25876548/c73999d1-ef90-453b-bf8a-f3f4e12b7dc3">

</details>

cc: @luacmartins @grgia

**Comment by Kicu** [comment]
Thank you @sobitneupane 

> Only improvement I could see is making use of existing logo prop if possible

For now I decided that separate prop makes more sense, just because the flow of svg asset has to be so different. One problem is that if `logo` prop will be a string then if its a URL pointing to a raster image we use `Image` but if it will be pointing to an `.svg` then a different component needs to be used.

I decided 2 props are perhaps a bit cleaner (more verbose) than a lot of "magical" if-statements 😅 I'd leave this as is for now if you think its not too terrible.

---
@luacmartins @grgia what would the next steps with this? Should I update my PR to Expensify/App to cover every case of our QR code avatars? 
What is the long term solution for the library, do you guys want to do your own fork? I don't think `patch-package` is a good solution for this as the changes span multiple files

**Comment by sobitneupane** [comment]
@Kicu I think we will need some styling changes. We should add some margin around `logoSVG` and decrease it's size to match it with `logo`.
<img width="76" alt="Screenshot 2024-03-28 at 17 31 45" src="https://github.com/Expensify/App/assets/25876548/1e663d9a-82f5-470f-9c6d-9938304c0264">      <img width="76" alt="Screenshot 2024-03-28 at 17 31 31" src="https://github.com/Expensify/App/assets/25876548/786249a4-ca8a-4bf0-96e8-3d90be0bc7e5">

**Comment by trjExpensify** [comment]
https://github.com/Expensify/App/issues/27288#issuecomment-2024760317 - @luacmartins @grgia 

>@luacmartins @grgia what would the next steps with this? Should I update my PR to Expensify/App to cover every case of our QR code avatars?
>What is the long term solution for the library, do you guys want to do your own fork? I don't think patch-package is a good solution for this as the changes span multiple files

**Comment by luacmartins** [comment]
Thanks for the ping, I had missed that comment. I [posted in Slack](https://expensify.slack.com/archives/C04878MDF34/p1712075138840339?thread_ts=1712069868.579899&cid=C04878MDF34) asking for more opinions on whether we should fork this lib.

**Comment by grgia** [comment]
Bumping in email again 🤞

**Comment by sobitneupane** [comment]
@luacmartins Should we wait for the fork and make [these changes](https://github.com/Kicu/react-native-qrcode-svg/pull/1) there or can we go ahead and merge this [PR](https://github.com/Expensify/App/pull/39072)?

**Comment by Kicu** [comment]
@luacmartins sure thing, will try to update everything today and ping you guys when done.
Also some extra work was done by us to improve this qr project, and a collague @war-in opened a PR with an example app written for this library, so that you can test QR generated in isolation.
If you're interested you can take a look: https://github.com/Kicu/react-native-qrcode-svg/pull/2

**Comment by luacmartins** [comment]
@Kicu here's the new fork - https://github.com/Expensify/react-native-qrcode-svg/. Could you try opening a PR on that fork? Let me know if you have any isseus

**Comment by Kicu** [comment]
@luacmartins thanks, it seems I don't have access to push to this repo:
```sh
mateusz@~/swmansion/projects/react-native-qrcode-svg % git remote -v
expensify-fork	git@github.com:Expensify/react-native-qrcode-svg.git (fetch)
expensify-fork	git@github.com:Expensify/react-native-qrcode-svg.git (push)
origin	git@github.com:Kicu/react-native-qrcode-svg.git (fetch)
origin	git@github.com:Kicu/react-native-qrcode-svg.git (push)

mateusz@~/swmansion/projects/react-native-qrcode-svg % git push expensify-fork mateusz/add-svg-logo 
ERROR: Permission to Expensify/react-native-qrcode-svg.git denied to Kicu.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Please take a look.
But if its problematic to give me access then I can just fork your fork I guess, and do the PR from fork.

**Comment by luacmartins** [comment]
Thanks for letting me know @Kicu. I've bumped our infra team member. I'll let you know if you should fork or not soon.

**Comment by luacmartins** [comment]
@Kicu could you fork our fork and then open a PR from your fork? I think that's usually the flow we have for other repos.

**Comment by Kicu** [comment]
@luacmartins @sobitneupane my opened to the fork: https://github.com/Expensify/react-native-qrcode-svg/pull/1

**Comment by Kicu** [comment]
I will push this task forward this week. I got involved in Search v1 which had priority so this was left a bit forgotten

**Comment by abekkala** [comment]
@Kicu can you provide an update

**Comment by Kicu** [comment]
@abekkala hey sorry for the delays here.

I intend to push and finish this but I have not yet started. Partial reason is because we had some work on avatars recently (parts of which Im doing). Some changed the logic of where their source comes from (purely tech changes) but also how they are styled (for workspace avatars, check this: https://github.com/Expensify/App/pull/39637).

There also was a hotfix merged and I have an open PR refactoring avatar code.
Before returning to work on this task I needed to wait until all other changes are merged.

I expect to implement this starting next week and it should not be a lot of work. Feel free to ask any additional questions if something is unclear.


**tl;dr**
We had a lot of refactor around avatars recently - better to finish them all before implementing this issue.

**Comment by Kicu** [comment]
@abekkala hey, first of all sorry it's taking so long.
However please keep in mind that for about 1.5 week of the last 2 there was a merge freeze, which means a lot of code was sitting and waiting to be merged, which slowed things down.

Status is:
this PR https://github.com/Expensify/App/pull/39072 mostly solves the problem but its not yet fully up to date, but is very close to being done. 
Specifically I was waiting for another one of my PRs (avatar related) to be merged: https://github.com/Expensify/App/pull/41846 - and that one was finally merged just yesterday.

So now I need 1-2 days to bring back the #39072 and test it. Since we have holidays on Thu and Fri this week, the PR will be ready for review either today by eod or on Monday.
Hope that clears it up.

**Comment by luacmartins** [comment]
Thanks for the update @Kicu. I'd still focus on the Search sorting PR since that's a higher priority, but let's get this one done soon after.

**Comment by Kicu** [comment]
**Status update:**

 - sorry this is taking so long - I'm doing this inbetween Search and react concurrent mode tasks, which have a higher prio
 - the current status is that the PR and code is ready for this, however there is a bug with SVG displaying in ios, which makes the logo display badly or not at all
 - I have discovered that the problem lies in one of our dependencies - `react-native-svg` which is the lib that we use to render SVGs. After update to new architecture/Fabric they have a small bug when re-drawing svg with another svg inside it; so temporarily this is a blocker

**Next steps:**
This week I plan to prepare a minimal reproduction of the bug and publish it to https://github.com/software-mansion/react-native-svg
I also should be able to actually create a PR with the fix, because I have some help from the maintainer of the library. After this we will still need to wait for a new version, and then my original PR can move forward.

**Comment by luacmartins** [comment]
Thanks for the update @Kicu!

**Comment by luacmartins** [comment]
Still being worked on

**Comment by Kicu** [comment]
can someone please reopen 😅 I started to work on a fix to `react-native-svg`, and once the fix is done the can move forward with the original PR

**Comment by luacmartins** [comment]
We still need this

**Comment by Kicu** [comment]
Update:
Currently I'm waiting for 2 PRs to be merged in `react-native-svg`:
 - https://github.com/software-mansion/react-native-svg/pull/2318 (merged)
 - https://github.com/software-mansion/react-native-svg/pull/2327

After these both are merged we will have to update the version of `react-native-svg` in the project, and then I can move forward with original task, which hopefully is already working. 
Will update the PR soon

**Comment by roryabraham** [comment]
Both those PRs were published in react-native-svg [v15.4.0](https://github.com/software-mansion/react-native-svg/releases/tag/v15.4.0), so we should be ready to move onto the next step of upgrading react-native-svg in E/App

**Comment by Kicu** [comment]
Correct. I've bumped it here in my PR for this issue: https://github.com/Expensify/App/pull/39072/files

There are no big breaking changes and I verified that all usages of svg in our App work. The only BC on v15 was bumping the version of supported ios. Otherwise smooth update, and is used in this feature.

**Comment by Kicu** [comment]
The issue was solved by the PR merged and there and there was 1 follow up created for this here: https://github.com/Expensify/App/issues/45876

**Comment by sobitneupane** [comment]
This was deployed to production last week. https://github.com/Expensify/App/pull/39072#issuecomment-2244524942 This should be ready for payment.

**Comment by abekkala** [comment]
### PAYMENT SUMMARY FOR JULY 30

- Fix: @Kicu [swm - no upwork payment]
- PR Review: @sobitneupane [$250] payment via NewDot

**Comment by garrettmknight** [comment]
$250 approved for @sobitneupane based on [this summary](https://github.com/Expensify/App/issues/27288#issuecomment-2258782486).

**Comment by ashimsharma10** [comment]
@abekkala @garrettmknight , I'm the issue reporter . Haven't received payment.  Its an old time bug. 😄

**Comment by ashimsharma10** [comment]
Bump @garrettmknight . Thanks !

**Comment by garrettmknight** [comment]
@abekkala can you help @ashimsharma10 out?

**Comment by abekkala** [comment]
Ah yes - this was reported in September before we stopped the bug bounty. 
- $50 payout for @ashimsharma10 

@ashimsharma10 can you please link your Upwork profile, I was unable to find you!

**Comment by ashimsharma10** [comment]
@abekkala Thank you !
Find me here https://www.upwork.com/freelancers/~018a92cf13e1e88eed

**Comment by abekkala** [comment]
@ashimsharma10 [$50 Offer Sent](https://www.upwork.com/nx/wm/offer/104095049)

**Comment by abekkala** [comment]
@ashimsharma10 payment sent and contract ended - thank you! 🎉
